rlkit/core/rl_algorithm.py

```diff
@@ -159,6 +167,7 @@ def train(self):
         gt.reset()
         gt.set_def_unique(False)
         self._current_path_builder = PathBuilder()
+        # TODO: why do we need this to be a property of rl_algo?
```

Comment:
I believe this was a holdover from the original rlkit which collected data online (one transition, one gradient step). Since we collect data in batch mode, I think it's safe to get rid of this.

Reply:
I'm a little scared of touchign this now. I don't think at training, we handle rollouts ending correctly, since we stop collecting data in a task based on number of steps. We then reset the env to switch to a new task, implicitly terminating the episode. However, our code does not handle this case of rollout ending (since originally, the algo would just resume from where it left off).

Reply:
got rid of this when switched to using the sampler beb84ca